src/westpa/cli/tools/w_timings.py

```diff
@@ -0,0 +1,112 @@
+from datetime import timedelta
+
+import numpy as np
+
+from westpa.tools import (
+    WESTTool,
+    WESTDataReader,
+    IterRangeSelection,
+)
+
+
+# NumPy fixed time units (excludes nonlinear units 'Y and 'M')
+TIME_UNITS = ('W', 'D', 'h', 'm', 's', 'ms', 'us', 'ns', 'ps', 'fs', 'as')
```

Comment:
what about  `μs`? It's seemingly included in numpy.
https://numpy.org/devdocs/reference/arrays.datetime.html#datetime-units

Reply:
https://github.com/westpa/westpa/pull/503/commits/daeca95ba51afcb0c8c3b6931d5e5a5d5b5372d0